cpp/ps0000/problem solve 0.cpp

```diff
@@ -1,15 +1,16 @@
-#include <iostream>
+#include <iostream> // 2/2
 // اول غلطه كتب @ وهي مفروض #
 // تانى غلطة في كلمه "iostream.h  هي مفروض من غير علامتين تنصيص ومن غير .h
 using namespace std;
 //نسي يكتب الجمله دي
-int main()
+int main() // 1/1
 // كتب كلمه void 

```

Comment:
for `void` it's typeless keyword for function declaration and it's used when function don't take any parameters. Here we use `main` function with type `int` but without any parameters